src/wifipaf/WiFiPAFEndPoint.cpp

```diff
@@ -70,6 +70,9 @@
 #define WIFIPAF_CONFIG_IMMEDIATE_ACK_WINDOW_THRESHOLD 1
 
 #define WIFIPAF_ACK_SEND_TIMEOUT_MS 2500
+#define WIFIPAF_WAIT_RES_TIMEOUT_MS 1000
+// If the network resources is unavailable for a long time, drop the connection
+#define WIFIPAF_MAX_RES_BLOCK_PERIOD 60
```

Comment:
add unit of measurement to values (is this seconds, ticks, ms, usec or something else?)
prefer c++ style constants (constexpr with kUpperCamelCase naming) because those are type-safe.

Reply:
I will change it. It should be from the basic definition